src/engine/controls/bpmcontrol.cpp

```diff
@@ -87,6 +87,12 @@ BpmControl::BpmControl(const QString& group,
     connect(m_pTranslateBeatsLater, &ControlObject::valueChanged,
             this, &BpmControl::slotTranslateBeatsLater,
             Qt::DirectConnection);
+    m_pTranslateBeatsMove = new ControlEncoder(ConfigKey(group,
+            "beats_translate_move"), false);
```

Comment:
I think this will make clang-format happy
```suggestion
    m_pTranslateBeatsMove =
            new ControlEncoder(ConfigKey(group, "beats_translate_move"), false);
```